lib/argparse.js

```diff
@@ -0,0 +1,52 @@
+var search = require('./index').search;
```

Comment:
Can you name this file query.js? Its purpose is to provide a command line client for querying our search endpoint. Arg parsing is just a thing we need to do to get there.